README.md

```diff
@@ -132,14 +134,17 @@ Table of Contents
   * https://snap-ci.com/ - Free for public repositories, 1 build at the time
   * http://www.appveyor.com/ - CD service for Windows, free for Open Source
   * [Comparison of Continuous Integration services](https://github.com/ligurio/Continuous-Integration-services)
-  * https://saucelabs.com/ - CI with scalable testing for mobile and web apps, free for Open Source
```

Comment:
right, yeah - i am not keen on dupes but we can put it wherever you and OP feel is best, you're right either will work

Ignore that link for now, I'll merge on red if URLs not from this PR is causing hassles, if it stays broken I'll remove it